src/main/java/command/AddCommand.java

```diff
@@ -0,0 +1,39 @@
+package command;
```

Comment:
not sure whether you have to do this but the naming convention for packages said if its for a school project then the root name of the package should be your project name so maybe need to name the package starting with ip or light? I'm not fully sure though